CODEOWNERS

```diff
@@ -316,6 +316,7 @@ Kconfig*                                  @tejlmand
 /tests/benchmarks/multicore/idle/         @adamkondraciuk @nrfconnect/ncs-low-level-test
 /tests/benchmarks/i2c_endless/            @nrfconnect/ncs-low-level-test
 /tests/benchmarks/spi_endless/            @nrfconnect/ncs-low-level-test
+/tests/benchmarks/peripheral_stress_test/ @nrfconnect/ncs-low-level-test
```

Comment:
Shall I shorten peripheral_stress_test? F.e. peripheral_load

Reply:
The name should correspond with the goal of these tests.
IMO if changes introduce stress tests, this name of the directory is acceptable. 
@nordic-piks and @PerMac , what do you think about it?

Reply:
I am for "peripheral_load"

Reply:
Test name was changed to "peripheral_load"